examples/source/1.components/amp-recaptcha-input.html

```diff
@@ -1,7 +1,7 @@
 <!--
   ## Introduction
 
-  The [`amp-recaptcha-input`](https://amp.dev/documentation/components/reference/amp-recaptcha-input.html) component appends a [reCAPTCHA v3](https://developers.google.com/recaptcha/docs/v3) token to AMP form submissions. `amp-recaptcha-input` does this by creating an iframe to load the reCAPTCHA v3 api script using the provided site key, and calling `grecaptcha.execute` with the provided site key and action.
+  The [`amp-recaptcha-input`](https://amp.dev/documentation/components/amp-recaptcha-input.html) component appends a [reCAPTCHA v3](https://developers.google.com/recaptcha/docs/v3) token to AMP form submissions. `amp-recaptcha-input` does this by creating an iframe to load the reCAPTCHA v3 api script using the provided site key, and calling `grecaptcha.execute` with the provided site key and action.
```

Comment:
Did you by chance do a find-all on `components/reference` in examples source? Could be that we have this error somewhere else as well 🙂

Reply:
I certainly did

Reply:
Great, thanks!